docs/Contributing/pull-request.rst

```diff
@@ -0,0 +1,182 @@
+has not really talked about the Games repository, nor talked about Rulesets or Tilesets. In this scenario,
+there isn't any compiling going on. The Ruleset or Tileset editor is editing files and testing locally. The
+Longturn Games repository is effectively a repository of Rulesets.
+
+
+5. Push The Changes In A Commit To Origin
+=========================================
+
+Now the moment arrives, you have a fully completed set of changes you want to share! The next step is to
+evaluate what all has changed and build a commit. A commit is an organized set of changes all bundled up
+together. There are lots of ways or organize commits, for this set of instructions we are going to assume
+that all of your changes are going to be bundled into one commit. If you want to learn how to build more than
+one commit into a Pull Request then you can read about `git add <https://git-scm.com/docs/git-add>`_ and
+`git commit <https://git-scm.com/docs/git-commit>`_. Most of the work is handled with :file:`git add`.
+
+.. code-block:: rst
+
+  ~/GitHub/freeciv21$ git status
+  ~/GitHub/freeciv21$ git add --all
+  ~/GitHub/freeciv21$ git commit
+
+
+The :file:`git status` command is used to list out all the changes that :file:`git` has seen. Changed files,
+new files, moved files, etc. are all listed. This command comes in handy when you want to organize your Pull
+Request into more than one commit. It's also very useful to ensure that what you see as changed follows along
+with what you THINK you have actually changed. Sometimes in the course of editing you may inadvertantly made
+a change to another file that you didn't intend to actually change. If you find that you accidentially changed
+a file, you can use the :file:`git restore <file>` command. This is also shown on the :file:`git status`
+command output.
+
+Once everything looks good from a :file:`git status` perspective, then issue the last two commands. The last
+command, :file:`git commit` will open a text editor (in Debian based systems this is often Nano). Put a message
+at the bottom below all of the hashes ( # ) and then save.
+
+At this point you have a commit of changes that you need to push to your fork (e.g. Origin).
+
+
+6. Requesting A Pull Request From Origin To Upstream
+====================================================
+
+This is the last major step in the process. To push the commit to your fork, issue this command:
+
+.. code-block:: rst
+
+  ~/GitHub/freeciv21$ git push origin
+
+
+You will be prompted for your SSH passkey and then the changes in the branch you created in step 3 will be
+pushed to your fork. Once complete, you can open your fork in a web browser and GitHub will notice the
+change and prompt you to open a Pull Request. The complete steps are documented
+here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork.
```

Comment:
Make "documented here" a link? It doesn't look very good currently

Reply:
I tried that, but the length of the URI makes it impossible. Spinx complains a lot.